tests/test_jump.py

```diff
@@ -545,10 +550,11 @@ def test_inside_ellipes5():
 @pytest.mark.skip(" used for local testing")
 def test_flag_persist_groups():
 #   gdq = fits.getdata("persistgdq.fits")
-    gdq = np.zeros(shape=(2,2,2,2))
-    print(gdq.shape[0])
-    gdq = gdq[:, 0:10, :, :]
-    total_snowballs = flag_large_events(
+    gdq = fits.getdata("../../../notebooks/ingdq.fits")
```

Comment:
This should not be here.  The 'ingdq.fits' file should not be outputted by the pipeline.  And for testing, all files should be written to the temporary directory.

Reply:
The test is set to skip so I don't understand how it is a problem but I'll remove the test.